modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,280 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
+     *         <tr><td>{@code macOS.NSColor.systemRedColor}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code macOS.NSColor.systemTealColor}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code macOS.NSColor.systemYellowColor}</td><td>{@link Color}</td></tr>
+     *         <tr></tr>
+     *         <tr><th colspan="2" scope="colgroup">Linux</th></tr>
+     *         <tr><td>{@code GTK.theme_name}</td><td>{@link String}</td></tr>
+     *         <tr><td>{@code GTK.theme_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_selected_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_selected_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_selected_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_selected_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.borders}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.unfocused_borders}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.warning_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.error_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.success_color}</td><td>{@link Color}</td></tr>
+     *         <tr></tr>
+     *     </tbody>
+     * </table>
+     *
+     * @since 22
+     */
+    public interface Preferences extends ObservableMap<String, Object> {
+        /**
+         * The platform appearance, which specifies whether applications should use a light
+         * or dark color scheme. The value of this property is derived from the perceptual
+         * brightness of {@link #backgroundColorProperty() backgroundColor} in relation to
+         * {@link #foregroundColorProperty() foregroundColor} and defaults to {@link Appearance#LIGHT}
+         * if the platform does not report color preferences.
+         *
+         * @return the {@code appearance} property
+         */
+        ReadOnlyObjectProperty<Appearance> appearanceProperty();
+
+        /**
+         * Gets the value of the {@link #appearanceProperty() appearance} property.
+         *
+         * @return the platform appearance
+         */
```

Comment:
Please remove this javadoc block. Properties should only have docs on the xxxProperty method (or, for concrete classes, the private xxxx field), not on the setters and getters. This will allow javadoc to copy the information from the property, and generate cross-links.